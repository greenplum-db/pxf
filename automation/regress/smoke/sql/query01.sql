SELECT * FROM cars ORDER BY id;
