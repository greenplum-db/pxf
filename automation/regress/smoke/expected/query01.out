SELECT * FROM cars ORDER BY id;
 id | year |    make    |  model  | msrp  
----+------+------------+---------+-------
  1 | 2021 | Toyota     | Camry   | 25045
  2 | 2021 | Chevrolate | Equinox | 23800
  3 | 2021 | Kia        | Soul    | 17590
(3 rows)

